### ISSUE TITLE
Empty fill color and fill opacity for paths with fill and stroke operations with 1.22.*

### ISSUE THREAD

**Issue by jn-chrn** [pymupdf/PyMuPDF#2450]
## Describe the bug (mandatory)

On some files, when extracting the drawings using `Page.get_drawings`, the result dictionary for some paths contains invalid properties since the version `1.22.0`. They are set to `None`, instead of the actual value. This happens for `fill` and `fill_opacity` for our case. No such issue occurs with the version `1.21.1`.

## To Reproduce (mandatory)

By running the following script with a simple PDF file: [file.pdf](https://github.com/pymupdf/PyMuPDF/files/11654265/file.pdf)

```python
import fitz

document = fitz.open("file.pdf")
page = list(document.pages())[0]
(single_drawing,) = list(page.get_drawings())

print(single_drawing)
```

One gets the following dictionary as first path using `Page.get_drawings` with `1.21.1`:

```python
{
    "items": [("re", Rect(60.0, 70.0, 100.0, 100.0), 1)],
    "type": "fs",
    "stroke_opacity": 0.9966222047805786,
    "color": (0.9490200281143188, 0.5372549891471863, 0.7019609808921814),
    "width": 1.0,
    "lineCap": (0, 0, 0),
    "lineJoin": 1.0,
    "dashes": "[] 0",
    "rect": Rect(60.0, 70.0, 100.0, 100.0),
    "seqno": 1,
    "even_odd": False,
    "fill_opacity": 0.6905926465988159,
    "closePath": False,
    "fill": (0.9686269760131836, 0.549019992351532, 0.9215689897537231),
}

```

and the following dictionary with `1.22.3`:

```python
{
    "items": [("re", Rect(60.0, 70.0, 100.0, 100.0), 1)],
    "type": "fs",
    "stroke_opacity": 0.9966222047805786,
    "color": (0.9490200281143188, 0.5372549891471863, 0.7019609808921814),
    "width": 1.0,
    "lineCap": (0, 0, 0),
    "lineJoin": 1.0,
    "fill": None,  # <-- empty although the rectangle is filled
    "fill_opacity": None,  # <-- empty although the rectangle is filled
    "closePath": False,
    "dashes": "[] 0",
    "rect": Rect(60.0, 70.0, 100.0, 100.0),
    "layer": None,
    "seqno": 1,
    "even_odd": False,
}
```

## Your configuration (mandatory)
 - Operating system, potentially version and bitness

```
3.10.11 (main, Apr 20 2023, 19:02:41) [GCC 11.2.0] 
 linux 
```

  - Python version, bitness

```
PyMuPDF 1.21.1: Python bindings for the MuPDF 1.21.1 library.
Version date: 2022-12-13 00:00:01.
Built for Python 3.10 on linux (64-bit).
```

- PyMuPDF version, installation method (**wheel** or **generated** from source).
    * PyMuPDF version 1.21.1 and 1.22.3, both installed using pip (so **wheel**).

**Comment by JorjMcKie** [comment]
This is a duplicate of #2365

**Comment by julian-smith-artifex-com** [comment]
Fixed in 1.22.5.
